swc_registry/get_entry_info.py

```diff
@@ -64,7 +68,9 @@ def _swc_content(self):
     @property
     def _content(self):
         entries = self._swc_content
-        current_entry = entries.get(self.swc_id, {})
+        current_entry = entries.get(self.swc_id, None)
```

Comment:
By default it will be None
```suggestion
        current_entry = entries.get(self.swc_id)
```